hw/formal/data/common_fpv_cfg.hjson

```diff
@@ -0,0 +1,36 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  flow:             fpv
+  flow_makefile:    "{proj_root}/hw/formal/data/fpv.mk"
+
+  import_cfgs:      [// common server configuration for results upload
+                     "{proj_root}/hw/data/common_project_cfg.hjson",
+                     "{proj_root}/hw/formal/data/jaspergold.hjson"]
```

Comment:
"Common" cfg should ideally not contain tool-specific settings.

Reply:
Is this better?

```suggestion
                     "{proj_root}/hw/formal/data/{tool}/{tool}.hjson"]
```

Reply:
Thanks Sri, yes that works better.